plugins/attachment-resources/src/components/AttachmentList.svelte

```diff
@@ -29,12 +29,25 @@
 {#if attachments.length}
   <Scroller contentDirection={'horizontal'} horizontal gap={'gap-3'} scrollSnap>
     {#each attachments as attachment}
+      {#if attachment !== undefined && attachment.type !== 'application/link-preview'}
```

Comment:
Can we handle this inside `AttachmentPreview` so to not duplicate the code? Or you made this intentionally to have two lines of previews?

Reply:
Thanks, fixed.